partner_identification/__manifest__.py

```diff
@@ -11,19 +11,22 @@
 {
     'name': 'Partner Identification Numbers',
     'category': 'Customer Relationship Management',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'depends': [
+        'sales_team',
```

Comment:
Can't you put the settings view in another place under *Sales > Configuration* that does not require this dependency?

Not a blocker, though, but would be nice.